### ISSUE TITLE
IP clearance status

### ISSUE THREAD

**Issue by ralf0131** [apache/incubator-dubbo-spring-boot-project#428]
As of January 31st, 2019, the latest status is as follows: 

| contributor                                   | commits | Lines changed       | contribution | Employer         | ASF ICLA signed? | Apache Dubbo Committer? | Contribution since join ASF? |
| --------------------------------------------- | ------- | ------------------- | ------------ | ---------------- | ---------------- | ----------------------- | ---------------------------- |
| [mercyblitz](https://github.com/mercyblitz)   | 100     | 16,496 ++  5,617 -- | Too much     | Alibaba          | Yes              | Yes                     | No                           |
| [lovepoem](https://github.com/lovepoem)       | 12      | 253 ++  22 --       | Too much     | Weidian          | Yes              | Yes                     | Yes                          |
| [chickenlj](https://github.com/chickenlj)     | 7       | 105 ++  80 --       | Too much     | Alibaba          | Yes              | Yes                     | No                           |
| [viviennes7](https://github.com/viviennes7)   | 2       | 8 ++  12 --         | Trivial      |                  | No               | No                      | Yes                          |
| [biyuhao](https://github.com/biyuhao)         | 2       | 5 ++  6 --          | Trivial      |                  | Yes              | No                      | Yes                          |
| [ralf0131](https://github.com/ralf0131)       | 2       | 42 ++  41 --        | License Info | Alibaba          | Yes              | Yes                     | Yes                          |
| [dongxuwang](https://github.com/dongxuwang)   | 2       | 27 ++  27 --        | Typo fix     |                  | No               | No                      | Yes                          |
| [qct](https://github.com/qct)                 | 2       | 19 ++  16 --        | Trivial      |                  | No               | No                      | No                           |
| [zhiwsun](https://github.com/zhiwsun)         | 1       | 0 ++  1 --          | Trivial      |                  | No               | No                      | Yes                          |
| [cnLGMing](https://github.com/cnLGMing)       | 1       | 2 ++  2 --          | Trivial      |                  | No               | No                      | Yes                          |
| [liaozan](https://github.com/liaozan)         | 1       | 4 ++  2 --          | Trivial      |                  | No               | No                      | Yes                          |
| [gejun123456](https://github.com/gejun123456) | 1       | 1 ++  1 --          | Trivial      |                  | No               | No                      | Yes                          |
| [Aunus](https://github.com/Aunus)             | 1       | 4 ++  4 --          | Trivial      |                  | No               | No                      | Yes                          |
| [wr1ttenyu](https://github.com/wr1ttenyu)     | 1       | 1 ++  1 --          | Trivial      |                  | No               | No                      | Yes                          |
| [ahxiaoei](https://github.com/ahxiaoei)       | 1       | 4 ++  4 --          | Trivial      |                  | No               | No                      | Yes                          |
| [boliuxyz](https://github.com/boliuxyz)       | 1       | 2 ++  2 --          | Trivial      |                  | No               | No                      | Yes                          |
| [hengyunabc](https://github.com/hengyunabc)   | 1       | 1 ++  1 --          | Trivial      | Alibaba          | Yes              | No                      | Yes                          |
| [fuyongde](https://github.com/fuyongde)       | 1       | 1 ++  1 --          | Trivial      |                  | No               | No                      | Yes                          |
| [nzomkxia](https://github.com/nzomkxia)       | 1       | 2 ++  2 --          | Trivial      | Alibaba          | Yes              | Yes                     | Yes                          |
| [htynkn](https://github.com/htynkn)           | 1       | 5 ++  9 --          | Trivial      | Meituan-Dianping | Yes              | Yes                     | No                           |

Most of the contributions are made after this project has been transferred to ASF.

There are 4 contributors who made contribution before the transfer. 3 of them are Dubbo committers. The only exception is:

* https://github.com/qct

Since the contribution is trivial, I think the ICLA is not required.

In conclusion, I think the IP clearance process is completed and there is nothing we need to do.
